packages/nodejs/grpc-migration.md

```diff
@@ -0,0 +1,14 @@
+# Migration guide from GRPCv1 to v2:
+
+## general:
+blockhash inputs are now optional, and if given must be given as a Uint8Array/buffer. If not given it uses the last final block.
+
+## getNextAccountNonce
+Renamed to getNextAccountSequenceNumber
+nonce field is renamed to sequenceNumber
+
+## getCryptographicParameters
+nothing?
+
+## getAccountInfo
+???
```

Comment:
Remembered we would need this, so I started it,
@rasmus-kirk please fill in the blanks/fix any errors.

Reply:
`getAccountInfo()` now also accepts an accountIndex, should this be in the migration guide? I figure not since it's just added functionality.